assets/js/queue_priv.js

```diff
@@ -0,0 +1,104 @@
+var REFRESH_SEC = 15,
+	displayDate = new Date(),
+	monthStrings = [ 'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec' ],
```

Comment:
The amount of code duplication between this js and the queue.js is really unfortunate, is there any way we could have a shared js for anything that is shared between them, and then separate files for the things that are different? Or is that not gonna be able to work in this instance?